src/main/java/org/opengis/cite/ogcapiedr10/queries/PositionQueryProcessor.java

```diff
@@ -27,7 +34,7 @@ public String validatePositionQueryUsingParameters(Set<String> collectionIds, St
             String url = JsonUtils.getCollectionURL(rootUri, collectionId);
 
             Response response = JsonUtils.getCollectionResponse(rootUri, collectionId, ini);
-            JsonPath jsonResponse = response.jsonPath();
+            JsonPath jsonResponse = JsonPath.from(new java.io.File("D:\\52n\\Projekte\\CITE Consulting\\edr-issue-133-collection-response.json"));
```

Comment:
It seems you used a local file path for debugging. Could you remove it and revert to the original logic?

Reply:
Good work spotting this. I reverted the changes. Thanks!